cypress/e2e/1-auth/sign-up.cy.ts

```diff
@@ -36,3 +36,11 @@ describe("Sign Up", () => {
     cy.get('[data-testid="email-error"]').should("be.visible");
   });
 });
+
+describe.only("check redirects", () => {
+  it("check user redirected from sing in to sign up page", () => {
+    cy.visit("/sign-in")
+    cy.contains('Create Account').click()
```

Comment:
Did you consider selecting an element by `data-testid` attribute? In the future, we might change the title in this button and this will break tests